bin/git-unsaved.js

```diff
@@ -39,7 +39,11 @@ new Tilda(`${__dirname}/../package.json`, {
     gitUnsaved(path, (err, data) => {
         spinner.clear();
         if (err) {
-            console.log(`${colorIt(`${err.message}`).red()}`);
+            if (err.message == 'stdout maxBuffer exceeded') {
+              console.log(`${colorIt(`${err.message} at ${path}`).red()}`);
```

Comment:
I think we can use this always, without an `if-else`. `err.message` and the `path` will exist always.